app/js/view/details_view.js

```diff
@@ -35,14 +39,75 @@ export default class DetailsView extends View {
     });
   }
 
+  filterNonAffectedApps(apps, filters) {
+    return apps.filter(app => {
+      return filters.reduce((prev, cur) => {
+        return prev || cur.test(app.manifestURL);
+      }, false);
+    });
+  }
+
+  populateAffectedApps(details) {
+    Promise.all([
+      AppsHelper.getAllApps(),
+      ManifestHelper.getManifest(details.manifestURL)
+    ])
+
+    .then(results => {
+      var apps = results[0];
+      var manifest = results[1];
+      var filters = manifest.customizations.map(customization => {
+        if (!customization.filter) {
+          console.warn('No filter provided in customization', customization);
+          customization.filter = '';
+        }
+        return new RegExp(customization.filter);
+      });
```

Comment:
Customizations is an array of objects, each one of which can (should) have a filter. Looking at the gecko code, it looks like there is no validation before creating the regex, so I'll just remove that check.